src/orb.yaml

```diff
@@ -0,0 +1,156 @@
+---
+version: 2.1
+
+definitions:
+  port-parameter: &port-parameter
+    description: port
+    type: integer
+  localhost-parameter: &localhost-parameter
+    description: host
+    type: string
+    default: localhost
+  wait-for-attributes: &wait-for-attributes
+    seconds_between_retries: <<parameters.seconds_between_retries>>
+    timeout: <<parameters.timeout>>
+    description: <<parameters.description>>
+
+description: Waits for things
```

Comment:
Maybe we can change it to something like:
*Wait until the execution of the command argument returns success(0) or until timeout*

Reply:
Changed to _Waits for things testable with a sh command_